src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:last-child a {
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main-text-color;
+    margin-right: 20px;
+    transition: color 0.3s;
+
+    &:hover {
+      color: $blue-brand-color;
```

Comment:
The variable `$blue-brand-color` is used for the hover color. Ensure that this variable is defined elsewhere in your SCSS files, otherwise it will cause a compilation error.